### PR TITLE
Add administrative namespace docs

### PR DESCRIPTION
**Pull request by biazmoreira**

### CODE REVIEW

**Review by ccapurso — APPROVED** [review]
Looks great! You made it very easy to review with the partial usage.

File: website/content/api-docs/system/auth.mdx [hunk 1]
```diff
@@ -256,7 +264,8 @@ $ curl \
 
 ## Read auth method tuning
 
-This endpoint reads the given auth path's configuration. _This endpoint requires
```
Comment (maxb) [inline website/content/api-docs/system/auth.mdx:259]: A line has been erroneously deleted here

**Review by divyaac — APPROVED** [review]

**Review by schavis — CHANGES_REQUESTED** [review]
Looks good overall. Made some suggestions to address styleguide issues.

File: website/content/api-docs/system/auth.mdx [hunk 2]
```diff
@@ -256,7 +264,9 @@ $ curl \
...
+@include 'alerts/restricted-admin.mdx'
+
+- This endpoint reads the given auth path's configuration. _This endpoint requires
```
Comment (schavis) [inline website/content/api-docs/system/auth.mdx:269]:
```suggestion
- This endpoint reads the given auth path's configuration. This endpoint requires
```

**Pull request merged**